app/src/main/java/org/wikipedia/notifications/NotificationFilterItemView.kt

```diff
@@ -70,10 +71,16 @@ class NotificationFilterItemView constructor(context: Context, attrs: AttributeS
     }
 
     fun setSingleLabel(text: String) {
-        binding.notificationFilterLanguageCode.visibility = View.INVISIBLE
-        binding.notificationFilterWikiLogo.visibility = View.GONE
+        val textSize = 14f
+        binding.notificationFilterLanguageCode.visibility = View.GONE
+        binding.notificationFilterWikiLogo.visibility = View.VISIBLE
+        binding.notificationFilterWikiLogo.imageTintList =
+            ColorStateList.valueOf(ContextCompat.getColor(context, ResourceUtil.getThemedAttributeId(context, R.attr.colorAccent)))
```

Comment:
Can we have the `colorAccent` as a `val` so that we don't need to use the method twice?